Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM docker.io/library/node:18.16.0-alpine
+FROM docker.io/library/node:20.11.1-alpine
 
 # uncomment to fix build on MacOS Apple Silicon chip
-# RUN apk add --no-cache python3 make g++
-RUN apk add git
+RUN apk add --no-cache python3 make g++
+# RUN apk add git
```

Comment:
Is `git` no longer needed here? I remember we had problems in CI without it.

Reply:
I believe it was needed when we had some git dependencies in `package.json`.
While we're at it, WDYT about adding `python3 make g++` by default, even it's only needed for local development?

Reply:
Sounds OK to me.